### PR TITLE
Update distribution.yaml

### PR DESCRIPTION
**Pull request by smoker771**

### CODE REVIEW

**Comment by smoker771** [comment]
Unfortunately 'bloom-release' failed to make pull request, so I make them manually. Sorry

File: melodic/distribution.yaml [hunk 1]
```diff
@@ -5048,6 +5048,17 @@ repositories:
...
       version: master
     status: developed
+  marvelmind_nav:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/MarvelmindRobotics/marvelmind_nav-release.git
+      version: 1.0.10-1
+    source:
+      test_pull_requests: false
```
Comment (tfoote) [inline melodic/distribution.yaml:5058]: We can only support pull request builds on github due to using https://plugins.jenkins.io/ghprb/

**Review by tfoote — CHANGES_REQUESTED** [review]
One small fixup to remove the PR builds. 

I also noticed browsing your repository that you don't have the BSD headers in each of your source files. As it's a file based license all source files should have it at the top of the file.

**Comment by tfoote** [comment]
And I'd highly recommend debugging bloom opening PRs automatically. If you're using 2 factor you need to do a little bit extra to support it: https://wiki.ros.org/bloom/Tutorials/GithubManualAuthorization 

The automated PRs are much less prone to small typos that cause extra iterations over time. They also have better titles etc which makes them easier to review.

File: melodic/distribution.yaml (hunk 1 above)
Comment (smoker771) [reply melodic/distribution.yaml:5058]: I just added license headers to all source files in the source repository. Do I need to create new release and/or pull request? Thank you

Comment (clalancette) [reply melodic/distribution.yaml:5058]: Yes, that's the easiest thing to do.  Bloom will automatically reopen the PR for you.

**Pull request closed without merging**